streams/src/main/java/org/apache/kafka/streams/processor/internals/assignment/AssignorConfiguration.java

```diff
@@ -253,6 +253,21 @@ public TaskAssignor taskAssignor() {
         }
     }
 
+    public org.apache.kafka.streams.processor.assignment.TaskAssignor userTaskAssignor() {
```

Comment:
consider returning `Optional<...>` to indicate that `null` is a semantically valid result